dotnet-desktop-guide/net/winforms/printing/snippets/how-to-print-in-windows-forms-using-print-preview/csharp/Form1.cs

```diff
@@ -0,0 +1,75 @@
+using System.Drawing.Printing;
+
+namespace Sample_print_winform_print_preview
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        //<string_declaration>
+        // Declare a string to hold the entire document contents.
+        private string DocumentContents="";
+
+        // Declare a variable to hold the portion of the document that
+        // is not printed.
+        private string StringToPrint="";
```

Comment:
These shouldn't be title case. They should be `documentContents` and `stringToPrint`